redis/src/main/java/org/apache/zeppelin/redis/RedisInterpreter.java

```diff
@@ -0,0 +1,455 @@
+        COMMANDS.put(Protocol.Command.GEORADIUS, new GEORADIUSCommand(clientPool));
+
+        // Hash Commands
+
+        // HyperLogLog Commands
+        COMMANDS.put(Protocol.Command.PFADD, new PFADDCommand(clientPool));
+        COMMANDS.put(Protocol.Command.PFCOUNT, new PFCOUNTCommand(clientPool));
+        COMMANDS.put(Protocol.Command.PFMERGE, new PFMERGECommand(clientPool));
+
+        // Key Commands
+
+        // List Commands
+
+        // Set Commands
+
+        // Sorted Set Commands
+
+        // String Commands
+        COMMANDS.put(Protocol.Command.APPEND, new APPENDCommand(clientPool));
+        COMMANDS.put(Protocol.Command.BITCOUNT, new BITCOUNTCommand(clientPool));
+        COMMANDS.put(Protocol.Command.BITFIELD, new BITFIELDCommand(clientPool));
+        COMMANDS.put(Protocol.Command.BITOP, new BITOPCommand(clientPool));
+        COMMANDS.put(Protocol.Command.BITPOS, new BITPOSCommand(clientPool));
+        COMMANDS.put(Protocol.Command.DECRBY, new DECRBYCommand(clientPool));
+        COMMANDS.put(Protocol.Command.DECR, new DECRCommand(clientPool));
+        COMMANDS.put(Protocol.Command.GETBIT, new GETBITCommand(clientPool));
+        COMMANDS.put(Protocol.Command.GET, new GETCommand(clientPool));
+        COMMANDS.put(Protocol.Command.GETRANGE, new GETRANGECommand(clientPool));
+        COMMANDS.put(Protocol.Command.GETSET, new GETSETCommand(clientPool));
+        COMMANDS.put(Protocol.Command.INCRBY, new INCRBYCommand(clientPool));
+        COMMANDS.put(Protocol.Command.INCRBYFLOAT, new INCRBYFLOATCommand(clientPool));
+        COMMANDS.put(Protocol.Command.INCR, new INCRCommand(clientPool));
+        COMMANDS.put(Protocol.Command.MGET, new MGETCommand(clientPool));
+        COMMANDS.put(Protocol.Command.MSET, new MSETCommand(clientPool));
+        COMMANDS.put(Protocol.Command.MSETNX, new MSETNXCommand(clientPool));
+        COMMANDS.put(Protocol.Command.PSETEX, new PSETEXCommand(clientPool));
+        COMMANDS.put(Protocol.Command.SETBIT, new SETBITCommand(clientPool));
+        COMMANDS.put(Protocol.Command.SET, new SETCommand(clientPool));
+        COMMANDS.put(Protocol.Command.SETEX, new SETEXCommand(clientPool));
+        COMMANDS.put(Protocol.Command.SETNX, new SETNXCommand(clientPool));
+        COMMANDS.put(Protocol.Command.SETRANGE, new SETRANGECommand(clientPool));
+        COMMANDS.put(Protocol.Command.STRLEN, new STRLENCommand(clientPool));
+    }
+
+    @Override
+    public InterpreterResult interpret(String input, InterpreterContext context) {
+        String[] tokens = splitTokens(input);
+        String keyword = tokens[0];
+
+        InterpreterResult result;
+        Protocol.Command command = Protocol.Command.valueOf(keyword);
+        if (COMMANDS.containsKey(command)) {
+            result = COMMANDS.get(command).execute(tokens);
+        } else {
+            result = new InterpreterResult(ERROR, "keyword not found");
```

Comment:
Maybe we can provide a better error to the user like
`Keyword " + tokens + " is not supportted`?

What do you think?